<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddItemtPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Item</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="YLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Item Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteItemPushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Item</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="XLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ItemNameLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>A-Crane</string>
    </property>
   </widget>
   <widget class="QWidget" name="graphicsViewContainer" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>541</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>491</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </widget>
   <widget class="QRadioButton" name="RBFixed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixed</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="RBTowed">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Towed</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="ItemListTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wire Length</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="WireLengthSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <double>5000.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>185</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wired With</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboBoxWiredWith">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Item Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboBoxItemType">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Towed</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Source</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Buoy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Streamer</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>RBFixed</zorder>
   <zorder>label_4</zorder>
   <zorder>AddItemtPushButton</zorder>
   <zorder>YLineEdit</zorder>
   <zorder>ZLineEdit</zorder>
   <zorder>label_2</zorder>
   <zorder>DeleteItemPushButton</zorder>
   <zorder>label_5</zorder>
   <zorder>label_3</zorder>
   <zorder>XLineEdit</zorder>
   <zorder>ItemNameLineEdit</zorder>
   <zorder>graphicsViewContainer</zorder>
   <zorder>RBTowed</zorder>
   <zorder>ItemListTable</zorder>
   <zorder>label</zorder>
   <zorder>WireLengthSpinBox</zorder>
   <zorder>label_6</zorder>
   <zorder>ComboBoxWiredWith</zorder>
   <zorder>label_7</zorder>
   <zorder>ComboBoxItemType</zorder>
   <zorder>checkBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
